packages/survey-creator-core/src/components/tabs/translation.scss

```diff
@@ -35,6 +35,10 @@
       width: calcSize(5);
     }
 
+    .spg-table__cell:nth-of-type(2) {
+      width: calcSize(5);
```

Comment:
How it will work with UI scaling in V2?

Reply:
it is ok because base-unit includes scaling too